hypershift-operator/main.go

```diff
@@ -61,6 +66,10 @@ import (
 )
 
 func main() {
+	ctrl.SetLogger(zap.New(zap.UseDevMode(true), zap.JSONEncoder(func(o *zapcore.EncoderConfig) {
```

Comment:
why is this moved and them duplicated into the new command?

Reply:
Accidentally duplicated, thank you for catching! Removed it from the new command.